.gitignore

```diff
@@ -11,3 +11,6 @@ gh-pages-generated/
 *.iml
 .idea/*
 !.idea/dictionaries/
+# gradle-profiler
+gradle-user-home
```

Comment:
It'd be better to avoid this issue but not ignore
https://github.com/avito-tech/avito-android/pull/1052/files#r646526650